src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -1614,10 +1615,8 @@ private static async Task ApplyBuildArgumentsAsync(Container dcpContainerResourc
                 var valueString = buildSecret.Value switch
                 {
                     FileInfo filePath => filePath.FullName,
-                    string stringValue => stringValue,
```

Comment:
?

Reply:
I was thinking ... wait ... I thought I just deleted that code ... and I did :)